frontend/appflowy_flutter/lib/plugins/database/widgets/card/card_bloc.dart

```diff
@@ -116,33 +117,54 @@ List<CellContext> _makeCells(
         !(fieldInfo.visibility?.isVisibleState() ?? false) ||
         (groupFieldId != null && cellContext.fieldId == groupFieldId);
   });
-  return cellContexts.toList();
+  return cellContexts
+      .map(
+        (cellCtx) => CellMeta(
+          fieldId: cellCtx.fieldId,
+          rowId: cellCtx.rowId,
+          fieldType: fieldController.getField(cellCtx.fieldId)!.fieldType,
```

Comment:
A future improvement: We are force unwrapping the value from `getField` everywhere. This doesn't make sense. If we are sure the value must not be null, we should change the return type of the `getField` function.